frontend/public/locales/en/translation.json

```diff
@@ -1123,6 +1123,7 @@
   "Delete storage class mapping": "Delete storage class mapping",
   "Delete subscription": "Delete subscription",
   "Delete VirtualMachine": "Delete VirtualMachine",
+  "Delete virtualmachinesnapshot": "Delete virtualmachinesnapshot",
```

Comment:
Seems like a strange casing for this message... should it be "virtual machine snapshot", "VirtualMachine snapshot", or "VirtualMachineSnapshot"?